backend-rust/migrations/0001_initialize.up.sql

```diff
@@ -312,7 +312,27 @@ CREATE TABLE bakers(
     -- Fraction of finalization rewards charged by the pool owner.
     -- Stored as a fraction of an amount with a precision of `1/100_000`.
     finalization_commission
+        BIGINT,
+    -- Transaction used for self suspending.
+    -- This is not null only when a baker have send the transaction for self suspending.
+    -- This should always be null, when `inactive_suspended` or `primed_for_suspension` column is
+    -- not null.
+    self_suspended
         BIGINT
+        REFERENCES transactions,
+    -- Block which suspended this baker due to inactivity.
+    -- This is not null only when a baker got suspended due to inactivity.
+    -- This should always be null, when `self_suspended` or `primed_for_suspension` column is not
+    -- null.
+    inactive_suspended
+        BIGINT
+        REFERENCES blocks,
+    -- Block which is primed for suspension this baker due to inactivity.
+    -- This is not null only when a baker got primed for suspension due to inactivity.
+    -- This should always be null, when `self_suspended` or `inactive_suspended` column is not null.
```

Comment:
See other comments, but I think it's OK for both this and `self_suspended` to be present.